scripts/admin/publish-meteor-tool.bat

```diff
@@ -0,0 +1,16 @@
+REM This file is copied line by line by the publish-meteor-tool-on-arch.sh script
+IF EXIST C:\tmp ( rmdir /s /q C:\tmp )
+md C:\tmp
+cd C:\tmp
+C:\git\bin\git.exe config --global core.autocrlf input
+C:\git\bin\git.exe clone https://github.com/meteor/meteor.git
+cd meteor
+C:\git\bin\git.exe fetch --tags
+C:\git\bin\git.exe checkout $GITSHA
+cd C:\tmp\meteor\packages\meteor-tool
+set METEOR_SESSION_FILE=C:\meteor-session
+C:\git\bin\curl -L http://downloads.sourceforge.net/sevenzip/7z920.msi ^> C:\7z.msi
+msiexec /i C:\7z.msi /quiet /qn /norestart
+set PATH=^%PATH^%;"C:\Program Files\7-Zip"
```

Comment:
it is a pithy we cannot put a comment here explaining why the heck we escape `%` in `^%PATH^%`